scripts/ansible/roles/windows/jenkins/tasks/environment-setup.yml

```diff
@@ -0,0 +1,81 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+---
+- name: Download all the installers
+  win_get_url:
+    url: "{{ item.value.url }}"
+    dest: "{{ item.value.dest }}"
+    timeout: 60
+  retries: 5
+  with_dict: "{{ packages }}"
+
+- name: Oracle JDK8 - Download
+  win_get_url:
+    url: "{{ jdk8_url }}"
+    headers:
+      Cookie: "oraclelicense=accept-securebackup-cookie"
+    dest: "{{ tmp_dir }}\\jdk-windows-x64.exe"
+
+- name: Add all the packages to the machine PATH
+  win_path: 
+    elements: "{{ packages_env_path }}"
+
+- name: Oracle JDK8 - Install
+  raw: "{{ tmp_dir }}\\jdk-windows-x64.exe /s REBOOT=Disable REMOVEOUTOFDATEJRES=1"
+
+- name: 7Zip - Install
+  win_package:
+    path: "{{ packages['7z']['dest'] }}"
+
+- name: Microsoft Visual Studio 2017 - Install
+  raw: "{{ packages['vs_2017']['dest'] }} --quiet --wait --add Microsoft.VisualStudio.Workload.VCTools --add Microsoft.VisualStudio.Component.VC.CMake.Project --includeRecommended"
+
+- name: Clang 7 - Install
+  raw: "{{ packages['clang7']['dest'] }} /S"
+
+- name: Git - Install
+  raw: "{{ packages.git.dest }} /SILENT"
+
+- name: OCaml - Create new temp install directory  
+  block:    
+   - win_file:        
+       state: absent        
+       path: "{{ tmp_dir }}\\OCaml"    
+   - win_file:        
+       state: directory        
+       path: "{{ tmp_dir }}\\OCaml"
+
+- name: OCaml - Extract  
+  raw: '7z.exe x "{{ packages.ocaml.dest }}" -r -y -o"{{ tmp_dir }}\OCaml"'
+
+- name: OCaml - Install  
+  win_shell: |    
```

Comment:
Ditto. A couple of extra spaces to be removed.

Reply:
Fixed